test/test_files/exceptions/binder/empty_db_binder_error.test

```diff
@@ -0,0 +1,9 @@
+-GROUP EmptyDBBinderErrorTest
+-DATASET CSV empty
+
+--
+
+-CASE MissingQueryNode
```

Comment:
btw should we put this into binder_error.test and then split the file at some point?

Reply:
To produce the error, i need a fresh database without any nodetables. So i can't put this file into binder_error.test